server/service/src/print/label.rs

```diff
@@ -32,6 +32,9 @@ pub fn print_qr_code(
     let payload = format!(
         r#"
         ^XA
+        ^FX CI command parameters:
+        ^FX - encoding (28 = UTF-8)
+        ^CI28
```

Comment:
Should allow printing supporting more languages I think. Recommended [here](https://labelary.com/zpl.html).